visual-inspection/inspection-frontend/src/components/InspectImage.vue

```diff
@@ -2,7 +2,7 @@
   <div class="inspector">
     <MultiBounce v-if="!prediction"
                  v-bind:numberOfDots="3"/>
-    <p v-show="prediction && currentPrediction">{{ $t('answer', { object: prediction }) }}</p>
+    <p v-show="prediction && currentPrediction">{{ $t('answer1')}} {{this.probability}} {{ $t('answer2', { object: prediction })}}</p>
```

Comment:
Es sollte konfigurierbar sein, ob wir das anzeigen oder nicht.

@Matthias271828 kann dir da weiterhelfen, der hat eine ähnliche Konfigurationsoption gerade in PR #861 für die Sichtbarkeit des "Erklär-Buttons" implementiert.